src/SprykerEco/Yves/Computop/Form/AbstractSubForm.php

```diff
@@ -22,6 +22,16 @@ abstract class AbstractSubForm extends AbstractSubFormType implements SubFormInt
      */
     public const FIELD_URL = 'url';
 
+    /**
+     * @var string
+     */
+    protected const TEMPLATE_PATH_PATTERN = '%s/%s';
```

Comment:
it's forbidden to store patterns for `sprintf` at constant
please move this string to `sprintf` directly

Reply:
Fixed